index.md

```diff
@@ -47,825 +50,633 @@ The primary goals for this transition are:
+            },
+            "stackTrace": {
+              "type": "string",
+              "description": "Optional stack trace for debugging"
+            }
+          }
+        },
+        "metadata": {
+          "type": "object",
+          "description": "Additional metadata about the query",
+          "properties": {
+            "query": {
+              "type": "string",
+              "description": "The original query"
+            },
+            "database": {
+              "type": "string",
+              "description": "Database that was queried"
+            },
+            "userTables": {
+              "type": "array",
+              "description": "List of any user tables created for TAP_UPLOAD",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    }
 
 
-### 3.5 Error Handling Strategy
+### Job status for completed query example
 
-Error handling needs to be coordinated across multiple system layers:
+    {
+      "schemaVersion": "1.0",
+      "jobID": "uws-123",
+      "qservID": "qserv-123",
+      "timestamp": "2025-03-19T..",
+      "status": "COMPLETED",
+      "queryInfo": {
+        "startTime": "2025-03-18T..",
+        "endTime": "2025-03-19T..",
+        "duration": 214,
+        "totalChunks": 167,
+        "completedChunks": 167
+      },
+      "resultInfo": {
+        "totalRows": 1000,
+        "resultLocation": "https://bucket/results_uws123.xml",
+        "format": "votable",
+        "sizeBytes": 128456
+      },
+      "metadata": {
+        "query": "SELECT TOP 10 * FROM table",
+        "database": "dp1"
+      }
+    }
 
-QServ API Level:
+### Job status with error
 
-The QServ service will report errors through HTTP status codes and detailed error messages. For example, when a query fails due to syntax errors:
+    {
+      "schemaVersion": "1.0",
+      "jobID": "uws-123",
+      "qservID": "qserv-123",
+      "timestamp": "2025-03-19T..",
+      "status": "ERROR",
+      "queryInfo": {
+        "startTime": "2025-03-19T..",
+        "endTime": "2025-03-19T..",
+        "duration": 3,
+        "totalChunks": 3,
+        "completedChunks": 1
+      },
+      "errorInfo": {
+        "errorCode": "QSERR-1",
+        "errorMessage": "Syntax Error at line 1",
+        "stackTrace": "at QservQueryExecutor.executeQuery..."
+      },
+      "metadata": {
+        "query": "SELECT TOP 10 * FROM dp1.Table",
+        "database": "dp1",
+        "userTables": ["my_objects"]
+      }
+    }
 
-```
-{
-   "success": 0,
-   "error": "Syntax error in query",
-   "error_ext": {
-       "position": 45,
-       "message": "Unexpected token LIMIT"
-   }
-}
-```
+## 4. Kafka Topics Structure
 
-TAP Service Level:
+- run_query - Requests to start a query from TAP service towards QServ
+- delete_query - Requests to stop & delete a query from TAP service towards QServ
+- job_status - Update to job status from QServ towards TAP service 
```

Comment:
We agreed on the following names for the Kafka topics, so you can update the diagram too:
lsst.tap.job-run, lsst.tap.job-delete and lsst.tap.job-status the lsst.tap prefix is used as the namespace the these topics in Sasquatch.

Review topic names throughout the document.